docs/CONTRIBUTING.md

```diff
@@ -61,7 +61,11 @@ npm run add-dependency --template=video-token --package=twilio-video
 
 ### Adding environment variables
 
-Any variable you want the user to have to set should be added to the `.env` file in your template directory and should include a commented line before that explaining what the variable is about. Example:
+Function templates can use environment variables for deploy-specific secrets by adding them to the `.env` file in the root of your template. These are the fields that the user will be able to pre-set on the CodeExchange web app. See `Step 2` here: https://www.twilio.com/code-exchange/simple-sms-forwarding
```

Comment:
I think this might be the first mention of the Code Exchange in the documentation for this repo. Do you think the connection between Function Templates and the Code Exchange should be more clearly documented, perhaps in a stand-alone document, including details like this?

Reply:
I added a section for CodeExchange and linked a wiki doc that explains more about the connection, let me know if you think there should be more info!